experimental/sdg-tracker/src/config/homeGoals.json

```diff
@@ -0,0 +1,121 @@
+[
```

Comment:
Can you combine this with the existing `rootTopics.json`?

Reply:
+1  -- we could use this elsewhere too.

Reply:
done